openassessment/xblock/static/css/openassessment-ltr.css

```diff
@@ -1,3 +1,3 @@
itle{margin-bottom:5px;border-bottom:2px solid #d9d9d9;padding-bottom:5px}.wrapper--xblock .message .message__content{color:#414141}.wrapper--xblock .message .message__content p{margin-bottom:10px}.wrapper--xblock .message--error{background:#feeced}.wrapper--xblock .message--error .message__title{color:#414141;border-bottom-color:#b20610}.wrapper--xblock .message--warning{background:#fff9eb}.wrapper--xblock .message--warning .message__title{color:#414141;border-bottom-color:#ffc01f}.wrapper--xblock .message--complete,.wrapper--xblock .step--student-training .message--correct,.step--student-training .wrapper--xblock .message--correct{background:#ecfaec}.wrapper--xblock .message--complete .message__title,.wrapper--xblock .step--student-training .message--correct .message__title,.step--student-training .wrapper--xblock .message--correct .message__title{color:#414141;border-bottom-color:#008100}.wrapper--xblock .message--incomplete,.wrapper--xblock .step--student-training .message--incorrect,.step--student-training .wrapper--xblock .message--incorrect{background:#fff9eb}.wrapper--xblock .message--incomplete .message__title,.wrapper--xblock .step--student-training .message--incorrect .message__title,.step--student-training .wrapper--xblock .message--incorrect .message__title{color:#414141;border-bottom-color:#ffc01f}.wrapper--xblock .message--error{display:none;opacity:0.0}.wrapper--xblock .has--error .message--error{display:block;opacity:1.0}.wrapper--xblock .message--inline{padding:10px 20px;background:#5e5e5e}.wrapper--xblock .message--inline .message__title{margin-bottom:0;border:none;padding-bottom:0;color:#fff}.wrapper--xblock .message--inline.message--error{background:#b20610}.wrapper--xblock .message--inline.message--error .message__content{color:#fff}.wrapper--xblock .message--inline.message--warning{background:#ffc01f}.wrapper--xblock .message--inline.message--warning .message__content{color:#111}.wrapper--xblock .message--inline.message--warning .message__title{color:#111}.wrapper--xblock .is--transitioning{padding:40px 40px;background:#f5f6f7}.wrapper--xblock .is--transitioning .transition__status{position:relative;top:50%;-webkit-transform:translateY(-50%);-moz-transform:translateY(-50%);-ms-transform:translateY(-50%);-o-transform:translateY(-50%);transform:translateY(-50%);text-align:center}.wrapper--xblock .is--transitioning .transition__status .wrapper--anim{display:block;-webkit-animation:pulse 3s ease-in-out infinite;-moz-animation:pulse 3s ease-in-out infinite;animation:pulse 3s ease-in-out infinite}.wrapper--xblock .is--transitioning .transition__status .icon,.wrapper--xblock .is--transitioning .transition__status .copy{display:block;color:#414141}.wrapper--xblock .is--transitioning .transition__status .icon{margin-bottom:10px}.wrapper--grid{*zoom:1;max-width:100%;margin-left:auto;margin-right:auto}.wrapper--grid:before,.wrapper--grid:after{content:" ";display:table}.wrapper--grid:after{clear:both}/*! Hint.css - v1.3.1 - 2013-11-23
```

Comment:
@rusrushal13 Are these changes part of your PR? If not, can you remove them?

Reply:
I have to run `make sass` to compile the `sass` files and they came from there as I changed a little bit of css